deployment/conf/.templates/setenv.sh.templ

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+
+CATALINA_HOME=/usr/share/tomcat8
+
+# The first existing directory is used for JAVA_HOME (if JAVA_HOME is not
+# defined in $DEFAULT)
+JDK_DIRS="/usr/lib/jvm/java-7-openjdk-amd64 /usr/lib/jvm/jre-7-oracle-x64"
```

Comment:
workspace is java 8 only

Reply:
I'll go strip these out - they are part of the standard Tomcat distro